web/src/constants/demos/otel-demo.proto.ts

```diff
@@ -0,0 +1,8 @@
pc Convert(CurrencyConversionRequest) returns (Money) {}\n}\n\n\nmessage Money {\n\n    string currency_code = 1;\n\n\n\n    int64 units = 2;\n\n\n\n\n\n\n\n    int32 nanos = 3;\n}\n\nmessage GetSupportedCurrenciesResponse {\n\n    repeated string currency_codes = 1;\n}\n\nmessage CurrencyConversionRequest {\n    Money from = 1;\n\n\n    string to_code = 2;\n}\n\n\n\nservice PaymentService {\n    rpc Charge(ChargeRequest) returns (ChargeResponse) {}\n}\n\nmessage CreditCardInfo {\n    string credit_card_number = 1;\n    int32 credit_card_cvv = 2;\n    int32 credit_card_expiration_year = 3;\n    int32 credit_card_expiration_month = 4;\n}\n\nmessage ChargeRequest {\n    Money amount = 1;\n    CreditCardInfo credit_card = 2;\n}\n\nmessage ChargeResponse {\n    string transaction_id = 1;\n}\n\n\n\nservice EmailService {\n    rpc SendOrderConfirmation(SendOrderConfirmationRequest) returns (Empty) {}\n}\n\nmessage OrderItem {\n    CartItem item = 1;\n    Money cost = 2;\n}\n\nmessage OrderResult {\n    string   order_id = 1;\n    string   shipping_tracking_id = 2;\n    Money shipping_cost = 3;\n    Address  shipping_address = 4;\n    repeated OrderItem items = 5;\n}\n\nmessage SendOrderConfirmationRequest {\n    string email = 1;\n    OrderResult order = 2;\n}\n\n\n\n\nservice CheckoutService {\n    rpc PlaceOrder(PlaceOrderRequest) returns (PlaceOrderResponse) {}\n}\n\nmessage PlaceOrderRequest {\n    string user_id = 1;\n    string user_currency = 2;\n\n    Address address = 3;\n    string email = 5;\n    CreditCardInfo credit_card = 6;\n}\n\nmessage PlaceOrderResponse {\n    OrderResult order = 1;\n}\n\n\n\nservice AdService {\n    rpc GetAds(AdRequest) returns (AdResponse) {}\n}\n\nmessage AdRequest {\n\n    repeated string context_keys = 1;\n}\n\nmessage AdResponse {\n    repeated Ad ads = 1;\n}\n\nmessage Ad {\n\n    string redirect_url = 1;\n\n\n    string text = 2;\n}\n\n\n\nservice FeatureFlagService {\n  rpc GetFlag(GetFlagRequest) returns (GetFlagResponse) {}\n  rpc CreateFlag(CreateFlagRequest) returns (CreateFlagResponse) {}\n  rpc UpdateFlag(UpdateFlagRequest) returns (UpdateFlagResponse) {}\n  rpc ListFlags(ListFlagsRequest) returns (ListFlagsResponse) {}\n  rpc DeleteFlag(DeleteFlagRequest) returns (DeleteFlagResponse) {}\n}\n\nmessage Flag {\n  string name = 1;\n  string description = 2;\n  bool enabled = 3;\n  google.protobuf.Timestamp created_at = 4;\n  google.protobuf.Timestamp updated_at = 5;\n}\n\nmessage GetFlagRequest {\n  string name = 1;\n}\n\nmessage GetFlagResponse {\n  Flag flag = 1;\n}\n\nmessage CreateFlagRequest {\n  string name = 1;\n  string description = 2;\n  bool enabled = 3;\n}\n\nmessage CreateFlagResponse {\n  Flag flag = 1;\n}\n\nmessage UpdateFlagRequest {\n  string name = 1;\n  bool enabled = 2;\n}\n\nmessage UpdateFlagResponse {}\n\nmessage ListFlagsRequest {}\n\nmessage ListFlagsResponse {\n  repeated Flag flag = 1;\n}\n\nmessage DeleteFlagRequest {\n  string name = 1;\n}\n\nmessage DeleteFlagResponse {}\n',
```

Comment:
We need to think about having this to be fetched from the actual repo in the future https://github.com/kubeshop/pokeshop/blob/master/api/protos/pokeshop.proto